docs/.vitepress/theme/posts.json

```diff
@@ -7,6 +7,7 @@
   {
     "url": "https://dev.to/jwilliamsr/the-transitory-nature-of-mlops-advocating-for-devopsmlops-coalescence-1k6j",
     "tags": ["ai", "devops", "open source", "machine learning"],
+    "author": "Jesse Williams",
```

Comment:
This post doesn't seem to have an `author` meta data so we are adding it manually here.